client/components/AutocompleteInput/components/PackageNameElement.tsx

```diff
@@ -0,0 +1,7 @@
+interface Props extends React.HTMLAttributes<HTMLElement> {
+  as?: 'h1'
```

Comment:
nit: If this is to only have one value, it is better modelled as a boolean.
`isHeading` (for e.g.)

Reply:
✅